src/mpl2/src/hier_rtlmp.cpp

```diff
@@ -317,6 +317,7 @@ void HierRTLMP::hierRTLMacroPlacer()
   //
   block_ = db_->getChip()->getBlock();
   dbu_ = db_->getTech()->getDbUnitsPerMicron();
+  manufacturing_unit_ = db_->getTech()->getManufacturingGrid();
```

Comment:
Though most LEF has this value it is possible for it to be unset.  You should check hasManufacturingGrid().  If none is present you could use 1dbu